nanome_vault/Vault.py

```diff
@@ -9,14 +9,10 @@
 from nanome.util.enums import NotificationTypes
 from nanome.api.structure import Complex
 
-from .VaultServer import VaultServer, EXTENSIONS
 from .menus import VaultMenu
 from . import VaultManager, Workspace
 
 DEFAULT_WEB_PORT = 80
```

Comment:
I noticed $PORT in deploy.sh. Do you want them to be independent?

Reply:
Yes, independent because the server always listens on 80/443. The deploy script and `-w` flag changes what is bound to the server and shows up in url string on plugin side.